kitchen.yml

```diff
@@ -0,0 +1,33 @@
+# Copyright 2018 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+---
+driver:
+  name: terraform
+  root_module_directory: test/fixtures
+
+provisioner:
+  name: terraform
+
+transport:
+  name: exec
+
+verifier:
```

Comment:
kitchen-terraform provides a `terraform` verifier that wraps `inspec`,  and adds a number of features that ease the integration of Terraform and InSpec. One feature is that the `terraform` verifier will automatically pass the Terraform outputs into InSpec as attributes, so you don't have to shuttle the values around yourself.

Switching from the inspec verifier to the terraform verifier should be pretty straight forward; the implementation at [project-factory #46](https://github.com/terraform-google-modules/terraform-google-project-factory/pull/46/files#diff-d3fbd21bc6be7ce4531c27a82b2619deR23) should be suitable for this with no modification.

While it isn't strictly necessary to use the `terraform` verifier, I'd recommend using it so that our modules are tested in a consistent manner.

Reply:
Good to know. Receiving your advice, I have updated a few things in kitchen.yml.
https://github.com/terraform-google-modules/terraform-google-pubsub/pull/1/commits/f92b9f85edc6b7b123e8954df5e8d4c285502968